mods/ra/tilesets/interior.yaml

```diff
@@ -1181,3 +1181,214 @@ Templates:
 		Categories: Floor
 		Tiles:
 			0: Clear
+	Template@291:
+		Id: 291
+		Images: lwal0001.int
+		Size: 1,1
+		PickAny: True
+		Categories: Wall
+		Tiles:
+			0: Clear
+			1: Clear
+			2: Clear
+			3: Clear
+	Template@292:
+		Id: 292
+		Images: lwal0002.int
+		Size: 1,1
+		PickAny: True
+		Categories: Wall
+		Tiles:
+			0: Wall
+			1: Clear
+			2: Clear
+			3: Clear
```

Comment:
This in combination with PickAny means there's a 25% chance the map editor will chose an impassable tile. Is that really intended?

Reply:
This is almost certainly wrong - there is no meaningful difference between the four artwork frames.

These are walls, so why are *any* of them marked as Clear?

Reply:
-.- copy paste error. Sorry.